src/components/Footer/Footer.module.scss

```
@@ -0,0 +1,183 @@
+@use '../../styles/variables.scss' as *;
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+$font-xs: 0.75rem; // 12px
+$font-sm: 0.875rem; // 14px
+$font-md: 1rem; // 16px
+$font-lg: 1.25rem; // 20px
+$font-xl: 1.5rem; // 24px
+$font-xxl: 2rem; // 32px
+$font-xxxl: 3rem; // 48px
+$font-xxxxl: 5rem; // 80px
+
+$space-xxss: 0rem; // 2px
+$space-xxs: 0.25rem; // 4px
+$space-xs: 0.5rem; // 8px
+$space-sm: 0.75rem; // 12px
+$space-md: 1rem; // 16px
+$space-lg: 1.5rem; // 24px
+$space-xl: 2rem; // 32px
+$space-xxl: 3rem; // 64px
+$space-xxxl: 4rem; // 48px
+$space-xxxxl: 6rem; // 96px
+
+.footer {
+  background-color: $verde-pistache;
+  padding: $space-md;
+  height: 25rem;
+
```

o que falta aqui e facilitaria a responsividade pra ti é adiconar o display flex column para celular e colocar o breakpoint de desktop pra fazer o display row para pc, coloca isso aqui nessa linha e roda pra ti ver:

```suggestion
display: flex;
  flex-direction: column;

  @include desktop {
    flex-direction: row;
    align-items: flex-start;
    gap: $space-lg;
  }
```